code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -67,16 +68,37 @@
 	if(candidates.len)
 		var/numTraitors = min(candidates.len, maxCount)
 
-		for(var/i = 0, i<numTraitors, i++)
+		for(var/i in 1 to numTraitors)
 			H = pick(candidates)
 			H.mind.make_Traitor()
 			candidates.Remove(H)
 
 		return TRUE
+	return FALSE
 
+/datum/admins/proc/makeObsessed(maxCount = 3)
+	//We're using the same list as traitors
+	var/datum/game_mode/traitor/temp = new
```

Comment:
This seems a bit *too* janky for me. Seems to only be used to fetch information on restricted roles and `age_check`, really feels unnecessary.

Feel free to get ike or otherwise somebody experienced to overrule this though.

Reply:
This file is janky.
I am using this to apply the same rules as traitor has, this means, excluding the same roles, having same age minimum etc.
Every other spawn works like that, sadly

Reply:
I don't care that much to rewrite the file I'll just remove mass obsessed